.github/workflows/build-images-for-tag-release.yaml

```
@@ -138,18 +138,41 @@ jobs:
         run: |
           # following sub-shells running make target should have yq already installed
           make yq
-      - name: Read operator bundle image reference
-        id: parsed-operator-bundle
+      - name: Set environment variables
+        id: set_env_variables
+        run: |
+          bash ./utils/release/load_github_envvar.sh
+      - name: Build catalog variables
```

maybe something like

```suggestion
- name: Set catalog image variables
```